core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -66,6 +69,17 @@ private Configs() {}
   public static final String DEFAULT_KV_ROCKSDB_BACKEND_PATH =
       String.join(File.separator, System.getenv("GRAVITINO_HOME"), "data", "rocksdb");
 
+  public static final String DEFAULT_RELATIONAL_JDBC_BACKEND_PATH =
+      String.join(File.separator, System.getenv("GRAVITINO_HOME"), "data", "jdbc");
```

Comment:
Is it better to use "h2" not "jdbc" for db path?

Reply:
I'm not very sure `h2` is a good name for the directory name as it's the default value of all JDBC database, assuming we are using SQLite as the database backend and using the default backend path, so the name is still `h2`, it's very puzzling.